scripts/docker_push.sh

```diff
@@ -0,0 +1,15 @@
+# This script attempts to build all the docker images, and then push then to the repository.
+# You'll need to have run `docker login` already;
+# check with scott.morrison@gmail.com for credentials.
```

Comment:
Maybe
```suggestion
# check with kim@tqft.net for credentials.
```
@kim-em Do you want your email here at all? Maybe just pointing to Zulip is enough?

Reply:
```suggestion
# check with Mathlib maintainers on Zulip for credentials.
```